Portfolio/Portfolio.js

```diff
@@ -1,4 +1,10 @@
+const linkToSignupPage = "../Signup/Signup.html";
+const linkToEditPage = "../PageEdit/PageEdit.html";
```

Comment:
Use PASCAL_CASE for your constants.

Reply:
Fixed. Thank you

Reply:
use ALL_CAPS for constants. In the previous comment i meant ALL_CAPS and mistakenly written pascal case however i wrote the requirement "PASCAL_CASE" which is actually all caps.